ibis/backends/duckdb/tests/test_geospatial.py

```diff
@@ -306,3 +306,13 @@ def test_literal_geospatial_inferred(con, shp, expected, snapshot):
 def test_load_geo_example(con):
     t = ibis.examples.zones.fetch(backend=con)
     assert t.geom.type().is_geospatial()
+
+
+def test_ops_geo_literals():
```

Comment:
Add a `con` here should fix your issue. Otherwise `.execute()` will instantiate a default backend.

```suggestion
def test_ops_geo_literals(con):
```